frontend/packages/noobaa-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -136,11 +135,9 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
           <BreadCrumbs
             breadcrumbs={[
               {
-                name: _.get(
-                  clusterServiceVersion,
-                  'spec.displayName',
+                name:
+                  ClusterServiceVersionModel?.[0]?.spec?.displayName ??
```

Comment:
The object is `clusterServiceVersion` not `ClusterServiceVersionModel`
``suggestion
                  clusterServiceVersion?.spec?.displayName ??
```